gradle.properties

```diff
@@ -23,7 +23,7 @@ systemProp.known_spark_binary_versions=3.3,3.4,3.5
 
 group=com.clickhouse
 
-clickhouse_jdbc_version=0.6.0
+clickhouse_jdbc_version=0.6.2
```

Comment:
upgrading clickhouse java client and adding test for clickhouse 24.3+ is irrelative to this PR's topic, they should happen in a dedicated PR